doc/array_conf/Arcconf_Conf.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Arcconf SmartArray Configuration Guide
+---
+LibStorageMgmt can manage Arcconf SmartArray via arcconf plugin
+(`libstoragemgmt-arcconf-plugin` package).
+You are also required to install `arcconf` binary tool from Microsemi
+website.
```

Comment:
The `arcconf` tool is also included in `Microsemi maxView Storage Manager` tool.